src/shared/icons/Close.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+type IconProps = {
```

Comment:
`TIconProps` - I think we should add it to our naming convention. 

what do you think @lucypoly ?